build/bin/sage-logger

```diff
@@ -61,15 +68,29 @@ else
     SED=cat
 fi
 
+if [ -z "$logfile" ]; then
+    # Just prefix, nothing else
+    ( exec 2>&1; sh -c "$cmd" ) | \
+        ( trap '' SIGINT; if [ -n "$GITHUB_ACTIONS" -a -n "$prefix" ]; then echo "::group::${logname}"; fi; $SED; if [ -n "$GITHUB_ACTIONS" -a -n "$prefix" ]; then echo "::endgroup::"; fi )
+
+    pipestatus=(${PIPESTATUS[*]})
+
+    if [ ${pipestatus[1]} -ne 0 ]; then
+        exit ${pipestatus[1]}
+    else
+        exit ${pipestatus[0]}
+    fi
+fi
+
 timefile="$logdir/$logname.time"
```

Comment:
Please a comment here saying `# if $logfille is given, then` or something, for readibility.

Reply:
Good idea, done in bfb292ae69a